admin-frontend/src/types/index.ts

```diff
@@ -19,3 +19,13 @@ export type UserInvite = {
   email: string;
   role: string;
 };
+
+export type Announcement = {
```

Comment:
My recent PR added a an IAnnouncement interface to types/announcements.ts.  I wonder if we can satisfy both our use cases with the same type or interface.  What do you think?

Reply:
Good idea, i reused the type you added